examples/list_hosts.py

```diff
@@ -20,57 +27,43 @@ def usage():
     print '-j|--json: Print output as json'
     print 'You can find your token at https://app.sysdigcloud.com/#/settings/user'
     sys.exit(1)
-
 try:
     opts, args = getopt.getopt(sys.argv[1:],"jd:",["json", "duration="])
 except getopt.GetoptError:
     usage()
-
-duration = 600
+duration = 3600
 print_json = False
 for opt, arg in opts:
     if opt in ("-d", "--duration"):
         duration = int(arg)
     elif opt in ("-j", "--json"):
         print_json = True
-
-if len(args) != 1:
-    usage()
-
 sdc_token = args[0]
-
-#
-# Instantiate the SDC client
-#
-sdclient = SdcClient(sdc_token)
-
 #
-# Prepare the query's metrics list. In this case, we have just one metric:
-# host.hostName. This is a 'key' metric, and we don't include any number metric.
-# Essentially, we create an 'enumeration' of hostnames.
 #
-metrics = [{"id": "host.hostName"}]
-
-#
-# Fire the query.
-# Note: there's no sampling time. This means that we're requesting the result to
-#       come as a single sample.
-#
-res = sdclient.get_data(metrics, # metrics list
-                        -duration,   # cover the last duration seconds...
-                        0,      # ... ending now...
-                        duration)    # ... with just one durations sample
-
-#
-# Show the results!
+# Prepare the query's metrics list.
+# In this case, we have one tag (used for segmentation) and one metric::
+# host.hostName. This is a tag, to identify each item of the output.
+# container.count: This is the metric
 #
+# Instantiate the SDC client
+sdclient = SdcClient(sdc_token)
+metrics = [{"id": "host.hostName"}, {"id": "container.count","aggregations":{"time":"avg","group":"avg"}}]
```

Comment:
I would write this list in multiple lines, so each metric we are querying is clearer:
```python
metrics = [
    {"id": "host.hostName"},
    {
        "id": "container.count",
        "aggregations":{"time":"avg","group":"avg"}
    },
]
```

Reply:
Very good idea.

One thing we should investigate is using a "prettier" configuration that (either manually or automatically) can enforce some guidelines like the one you suggested. As much as I like consistency, I don't like leaving the burden of enforcing consistency in the hands of every developer. Let's work on it outside the scope of this PR ;-)